src/elm/Questionnaires/Index/View.elm

```diff
@@ -130,12 +157,21 @@ listingActions wrapMsg appState questionnaire =
 
         editable =
             isEditable appState questionnaire
+
+        migrating =
+            questionnaire.state == Migrating
+
+        outdated =
+            questionnaire.state == Outdated
     in
     []
-        |> listInsertIf fillQuestionnaire editable
+        |> listInsertIf fillQuestionnaire (editable && not migrating)
         |> listInsertIf viewQuestionnaire (not editable)
```

Comment:
Maybe this one should also be restricted by migration? It is basically the same thing as fillQuestionnaire, except the questionnaire is rendered with disabled inputs.